tests/cases/fourslash/importNameCodeFix_importType10.ts

```diff
@@ -0,0 +1,27 @@
+/// <reference path="fourslash.ts" />
+
+// @module: es2015
+
+// sorting and multiline imports and trailing commas, oh my
+
+// @Filename: /exports.ts
+//// export interface SomeInterface {}
+//// export class SomePig {}
+
+// @Filename: /a.ts
+//// import {
+////     type SomeInterface,
+////     type SomePig,
+//// } from "./exports.js";
+//// new SomePig/**/
+
+goTo.marker("");
+verify.importFixAtPosition([
+`import {
+    type SomeInterface,
+    SomePig,
+} from "./exports.js";
+new SomePig`],
+    /*errorCode*/ undefined,
+    { organizeImportsTypeOrder: "inline" }

```

Comment:
It might be more interesting to make this one `last`, since that would require swapping the order of import specifiers after `type` gets removed from `SomePig`

Reply:
`tests/cases/fourslash/importNameCodeFix_importType7.ts` does this same test case (since last is the default behavior). I can add a note in that test that states that

Reply:
Though, now that I look at [importFixAtPosition](https://github.com/microsoft/TypeScript/blob/5a355093bb8c7d006966c3eb3aaf976cf4958517/src/harness/fourslashImpl.ts#L3523), it undoes the codefix so that we could make them into one test